### ISSUE TITLE
Create index files for codec files generated by ts-proto

### ISSUE THREAD

**Issue by willclarktech** [cosmos/cosmjs#647]
Manually/with a script/feature request to ts-proto.

**Comment by willclarktech** [comment]
@webmaster128 OK to close this?

**Comment by ethanfrey** [comment]
I agree. Other repos should not import codec, but compile what they need locally.
CosmJS cannot maintain all protobuf files possibly needed

**Comment by webmaster128** [comment]
I'm okay with avoiding any kind of intermediate solutions here. But I would still be happy to see index files being generated automatically. This would also be useful internally.

**Comment by willclarktech** [comment]
See https://github.com/stephenh/ts-proto/issues/212
